common/src/main/kotlin/flank/common/Files.kt

```diff
@@ -111,7 +111,7 @@ fun File.hasAllFiles(fileList: List<String>): Boolean {
 
 fun String.fileExists(): Boolean = Paths.get(this).exists()
 
-fun osPathSeperator() = (if (isWindows) "\\" else "/")
+fun osPathSeparator() = (if (isWindows) "\\" else "/")
```

Comment:
Am i blind what actually changed on this line?

Reply:
There was a typo `sepErator`. I've changed it to `sepArator`

Reply:
is it different from `File.pathSeparator` ?